src/components/virtual-list/virtual-list-class.js

```diff
@@ -238,7 +238,7 @@ class VirtualList extends Framework7Class {
         itemEl.f7VirtualListIndex = index;
       } else {
         if (vl.renderItem) {
-          vl.tempDomElement.innerHTML = vl.renderItem(items[i], index).trim();
+          vl.tempDomElement.innerHTML = vl.renderItem(items[i], { index: index } ).trim();
```

Comment:
This is a copy from Framework7 1.6.4

>                                 vl.tempDomElement.innerHTML = vl.template(items[i], {index: index}).trim();

don't know why the spaces fail in your build setup ?